functions/delete.py

```diff
@@ -1,8 +1,18 @@
 def delete(todos):
-    print("Enter number what you want to remove")
-    enter = int(input()) - 1
-    for i, k in enumerate(todos):
-        if i == enter:
-            del todos[k]
-            break
+    print("Enter the priority number what you want to remove")
+    enter = str(input())
```

Comment:
You can probably take the input as is, prompting user a value between h, m, and l so that you don't have to cast the variable to integer